apps/web/src/pages/api/forms/editStudy.ts

```diff
@@ -22,7 +22,7 @@ export default withApiHandler<ExtendedNextApiRequest>(Roles.SponsorContact, asyn
     assert(CPMS_API_USERNAME)
     assert(CPMS_API_PASSWORD)
 
-    const { studyId, plannedClosureDate, plannedOpeningDate, actualClosureDate, actualOpeningDate, ...studyData } =
+    const { id, studyId, plannedClosureDate, plannedOpeningDate, actualClosureDate, actualOpeningDate, ...studyData } =
```

Comment:
Why change from studyId to id?

Won't this invalidate existing links already emailed to sponsors?

Reply:
I don't believe this will have any impact. 

We are resolving which id we are trying to update on cpms and redirect to, and differentiate between the SE Study table Id and the cpms id/study id more clearly. 

The id in the link to the details and edit page remains unchanged, we are just referring to it differently in the code. We have been calling it StudyId, which is confused with the CPMS studyId/cpmsId. This id is the SE Study table record Id (`Study.Id`), not actually the study id

Reply:
I don't think it will invalidate anything, but I do think its confusing - we should use StudyId in reference to the SE Study.Id (as they do throughout other code) - Se has no concept of the StudyId being a thing in CPMS, so we should just refer to it as cpmsId